.gitignore

```diff
@@ -1,3 +1,4 @@
 extension.zip
 extension/dist
 node_modules
+package-lock.json
```

Comment:
npm shouldn't be used at all since this project uses yarn's version resolution, I think it's better to explicitly show the npm lockfile as untracked to remind contributors to use yarn

Reply:
That's fair. Will remove.